pkg/apis/pipeline/v1/pipelinerun_types.go

```diff
@@ -412,6 +412,9 @@ const (
 	PipelineRunReasonInvalidParamValue PipelineRunReason = "InvalidParamValue"
 )
 
+// PipelineTaskOnErrorAnnotation is used to pass the failure strategy to TaskRun pods
```

Comment:
nit: probably worth mentioning here that the failure strategy is from the pipelineTask's onError field

Reply:
SG! I have updated the commnet, PTAL